### PR TITLE
dbt-databricks migration doc

### PR DESCRIPTION
**Pull request by nghi-ly**
## Description & motivation

Docs review of the dbt-databricks migration page: https://docs.getdbt.com/guides/migration/tools/migrating-from-spark-to-databricks. Focus was on:

- being more scannable 
- conciseness 
- page layout, specifically the rightbar

### CODE REVIEW

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md [hunk 1]
```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
...
-### dbt Core
+</Tabs>
 
-In dbt Core, migrating to the dbt-databricks adapter from dbt-spark requires that you:
-1. install the new adapter in your environment, and
-2. modify your target in your `~/.dbt/profiles.yml`
+<!-- End tabs for dbt Cloud and dbt Core -->
 
-These changes will be needed for all users of your project.
+### Example
```
Comment (runleonarun) [inline website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: Just a tiny not. should we say "Examples?"

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md [hunk 2]
```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
...
 
-#### Example
+Below are examples of the `profiles.yml` file. It shows what authentication setup is like with `dbt-spark` and how much simpler is it with `dbt-databricks` when connecting to an SQL endpoint. A cluster example is also effectively the same.
```
Comment (runleonarun) [inline website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: Maybe we can use "the following" instead of "below" to make it more accessible.
```suggestion
You an use the following examples of the `profiles.yml` file to see the authentication setup with `dbt-spark` compared to the simpler setup with `dbt-databricks` when connecting to an SQL endpoint. A cluster example would look similar.
```

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md [hunk 3]
```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
...
 ---
 
+You can use the [dbt-databricks adapter](https://github.com/databricks/dbt-databricks) to establish a dbt connection with Databricks, instead of the `dbt-spark` adapter. In collaboration with dbt Labs, Databricks built this adapter using dbt-spark as the foundation and added some critical improvements. With it, you get an easier set up &mdash; requiring only three inputs for authentication &mdash; and more features through the Delta table format.
 
-## Pre-requisites
+## Simpler authentication
 
-In order to migrate to dbt-databricks, your project must be compatible with `dbt 1.0` or greater as dbt-databricks is not supported pre `dbt 1.0`. [This guide](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.0) will help you upgrade your project if necessary.
+Previously, you had to provide a `cluster` or `endpoint` ID which was hard to parse from the `http_path` that you were given. Now, it doesn't matter if you're using a cluster or an SQL endpoint because the [dbt-databricks setup](/reference/warehouse-setups/databricks-setup) requires the _same_ inputs for both. All you need to provide is:
+- the hostname of the Databricks workspace
```
Comment (runleonarun) [inline website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: I think we can remove "the" and "an" in the bullets without losing any meaning.

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md [hunk 4]
```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
...
+- For dbt Cloud, you need administrative (admin) privileges to migrate dbt projects.
 
-## Migration
-### dbt Cloud
+<!-- tabs for dbt Core and dbt Cloud -->
+<Tabs>
 
-#### Credentials
-If you are already successfully connected to Databricks using the dbt-spark ODBC method in dbt Cloud, then you have already supplied credentials in dbt Cloud to connect to your Databricks workspace. Each user will have added their Personal Access Token in their dbt Cloud profile for the given dbt project, which allows them to connect to Databricks in the dbt Cloud IDE, and additionally, an admin will have added an access token for each deployment environment, allowing for dbt Cloud to connect to Databricks during production jobs.
+<TabItem value="core" label="dbt Core">
```
Comment (runleonarun) [inline website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: This is a tiny nit, but can we make Cloud first? It is first alphabetically and it looks like the 1st tab is the selected by default, so I think that might be preferred

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md [hunk 5]
```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
...
 
-## Pre-requisites
+## Simpler authentication
 
-In order to migrate to dbt-databricks, your project must be compatible with `dbt 1.0` or greater as dbt-databricks is not supported pre `dbt 1.0`. [This guide](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.0) will help you upgrade your project if necessary.
+Previously, you had to provide a `cluster` or `endpoint` ID which was hard to parse from the `http_path` that you were given. Now, it doesn't matter if you're using a cluster or an SQL endpoint because the [dbt-databricks setup](/reference/warehouse-setups/databricks-setup) requires the _same_ inputs for both. All you need to provide is:
+- the hostname of the Databricks workspace
+- the HTTP path of the Databricks SQL warehouse or cluster
+- an appropriate credential
```
Comment (runleonarun) [inline website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: Should credential be plural? If it's just the token they need (singular) then maybe we should specify they need a token?

Edit: I see it's plural later, so let's make them the same!

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md [hunk 6]
```diff
@@ -3,77 +3,93 @@ title: "Migrating from dbt-spark to dbt-databricks"
...
+6. Select **Databricks (dbt-databricks)** for the adapter and enter the:
+    1. `hostname`
+    2. `http_path`
+    3. (optional) catalog name
+7. Click **Save**.
+
+Everyone in your organization who uses dbt Cloud must refresh the IDE before starting work again. It should refresh in less than a minute.
 
-After the above steps have been performed, all users will have to refresh their IDE before being able to start working again. It should complete in less than a minute. 
+#### Your credentials with the migration
```
Comment (runleonarun) [inline website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: I feel like this section should possibly talk about what you see and what you should do when the authentication doesn't work. I'm a little unclear what my takeaway should be here?

Comment (runleonarun) [reply website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: Also, should we link to the appropriate docs for how to authenticate if you can't? I think these might be the ones how to link a warehouses?

**Review by runleonarun — CHANGES_REQUESTED** [review]
@nghi-ly This is a huge improvement on the original doc! I really love the way you've reorganized it for readability and condensed what could be condensed (like the tabbed Core/Cloud 😍 ). 

I really just have some small fine tuning suggestions! Let me know if you have any questions.

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md (hunk 6 above)
Comment (nghi-ly) [reply website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: My takeaway when I was going through this content was: You do not need to reenter your credentials with the migration. 

Maybe I should start with that ^ then explain why credentials don't need to be inputted again. Lemme give that a try!

**Comment by nghi-ly** [comment]
@runleonarun : tysm for the review! great review comments. have updated the page. please take a look! imma dig into this merge conflict.

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md (hunk 6 above)
Comment (runleonarun) [reply website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md]: That works, Ly!

File: website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md [hunk 7]
```diff
@@ -3,78 +3,94 @@ title: "Migrating from dbt-spark to dbt-databricks"
...
+Everyone in your organization who uses dbt Cloud must refresh the IDE before starting work again. It should refresh in less than a minute.
+
+#### About your credentials
+
+When you update the Databricks connection in dbt Cloud for the migration, your team will not lose their credentials. This makes migrating easier since it only requires you to delete the Databricks connection and re-add the cluster or endpoint information.
+
+With a successful connection to Databricks using the `dbt-spark` ODBC method, the following credentials will not get lost:
 
-After the above steps have been performed, all users will have to refresh their IDE before being able to start working again. It should complete in less than a minute. 
+- The credentials you supplied to dbt Cloud to connect to your Databricks workspace.
```
Comment (nghi-ly) [inline website/docs/guides/migration/tools/migrating-from-spark-to-databricks.md:69]:
```suggestion
- The credentials you supplied in dbt Cloud to connect to your Databricks workspace.
```

**Review by runleonarun — DISMISSED** [review]
@ly this looks great!  Maybe @schottj and @Fleid make sure it looks good to them too.

**Comment by Fleid** [comment]
To be honest I liked having a **why** section (simpler auth, better defaults, pure python) followed by a **migrate** section.
But I also like the new overall flow better. 

So if I just want to learn about the topic, this new version is better. But if I'm actually here to migrate I may get confused about the actionability of the first part of the article (which can just be ignored when actually going through the process).

My 2 cents ;)

**Comment by nghi-ly** [comment]
@Fleid : lemme work on the first sentence and see if i can make it more clear about the migration

**Comment by nghi-ly** [comment]
@Fleid : updated the first sentence so:

- it refers to the "migration" to be a little more specific / actionable than "you can use" 
- there is a link to the migration steps subsection

**Review by runleonarun — APPROVED** [review]
@nghi-ly  This looks amazing! I think we can publish this and loop back with @Fleid after the new year and make sure we've addressed his concerns ❤️

**Comment by Fleid** [comment]
Perfect, thanks a lot :)
